tooling/nargo/src/foreign_calls/mod.rs

```diff
@@ -62,6 +66,20 @@ impl ForeignCall {
             _ => None,
         }
     }
+
+    /// Returns the foreign call status based on its name:
+    /// create_mock => 'mocked'
+    /// valid ForeignCall name => 'handled'
+    /// otherwise => 'unhandled'
+    pub(crate) fn check_oracle_status(name: &str) -> OracleResult {
+        if name == ForeignCall::CreateMock.name() {
+            OracleResult::Mocked
```

Comment:
Why aren't all the other [mock functions](https://github.com/noir-lang/noir/blob/14bb5ab228fcf0f995bac4b3a07773de6ae5b644/tooling/nargo/src/foreign_calls/mod.rs#L61-L65) considered to be `Mocked` status?

Reply:
I wanted to identify mocked oracles by looking at the create_mock() function, but it is not easy. Instead I  used an heuristic that consider that all oracles are mocked if there is one create_mock.